modules/installation-user-defined-tags-azure.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+// * installing/installing_azure/installing-azure-customizations.adoc
+
+:_content-type: CONCEPT
+[id="installing-azure-user-defined-tags_{context}"]
+= Configuring the user-defined tags for Azure
+
+In {product-title} you can use the tags for grouping resources and for managing resource access and cost. You can define the tags on the Azure resources in the `install-config.yaml` file only during {product-title} cluster creation. You cannot modify the user-defined tags after cluster creation.
+
+Support for user-defined tags is available only in {product-title} 4.13 as a Technology Preview (TP) and for the resources created in the Azure Public Cloud. User-defined tags are not supported for the {product-title} clusters upgraded to {product-title} 4.13.
+
+User-defined and {product-title} specific tags are applied only to the resources created by the {product-title} installer and its core operators such as Machine api provider azure Operator, Cluster Ingress Operator, Cluster Image Registry Operator.
+
+By default, {product-title} installer attaches the {product-title} tags to the Azure resources. These {product-title} tags are not accessible for the users.
+
+You can use the `.platform.azure.userTags` field in the `install-config.yaml` file to define the list of user-defined tags as shown in the following `install-config.yaml` file.
+
+.Sample `install-config.yaml` file
+[source,yaml]
+----
+additionalTrustBundlePolicy: Proxyonly <1>
+apiVersion: v1
+baseDomain: catchall.azure.devcluster.openshift.com <2>
+featureSet: TechPreviewNoUpgrade <3>
+compute: <4>
+- architecture: amd64
+  hyperthreading: Enabled <5>
+  name: worker
+  platform: {}
+  replicas: 3
+controlPlane:  <6> <7>
```

Comment:
Since both <6> and <7> are used only once, is there a reason why we have multiple callout numbers?

Reply:
My bad. Fixed the callout numbers 

Addressed